src/main/kotlin/no/skatteetaten/aurora/boober/feature/TopologyViewFeature.kt

```diff
@@ -0,0 +1,54 @@
+package no.skatteetaten.aurora.boober.feature
+
+import io.fabric8.openshift.api.model.DeploymentConfig
+import no.skatteetaten.aurora.boober.model.AuroraConfigFieldHandler
+import no.skatteetaten.aurora.boober.model.AuroraConfigFileType
+import no.skatteetaten.aurora.boober.model.AuroraContextCommand
+import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
+import no.skatteetaten.aurora.boober.model.AuroraResource
+import org.springframework.stereotype.Service
+
+@Service
+class TopologyViewFeature : Feature {
+
+    val topology = "topologyView"
+    val allowedFileTypes = setOf(
+        AuroraConfigFileType.BASE,
+        AuroraConfigFileType.BASE_OVERRIDE,
+        AuroraConfigFileType.APP,
+        AuroraConfigFileType.APP_OVERRIDE
+    )
+
+    override fun handlers(header: AuroraDeploymentSpec, cmd: AuroraContextCommand): Set<AuroraConfigFieldHandler> {
+        return setOf(
+            AuroraConfigFieldHandler(name = "$topology/group", allowedFilesTypes = allowedFileTypes),
+            AuroraConfigFieldHandler(name = "$topology/runtime", allowedFilesTypes = allowedFileTypes),
+            AuroraConfigFieldHandler(name = "$topology/connectsTo", allowedFilesTypes = allowedFileTypes)
+        )
+    }
+
+    override fun modify(adc: AuroraDeploymentSpec, resources: Set<AuroraResource>, context: FeatureContext) {
+        val group: String? = adc.getOrNull("$topology/group")
+        val runtime: String? = adc.getOrNull("$topology/runtime")
+        val connectsTo: List<String> = adc.getDelimitedStringOrArrayAsSet("$topology/connectsTo")
+            .filter { it.isNotBlank() }
+
+        resources
+            .filter { it.resource.kind == "DeploymentConfig" }
```

Comment:
skal endre her til å legge labels på alle ressurser ref. https://github.com/redhat-developer/app-labels/blob/master/labels-annotation-for-openshift.adoc#labels